odxtools/parameters/codedconstparameter.py

```diff
@@ -40,7 +40,7 @@ def _resolve_snrefs(self, diag_layer: "DiagLayer") -> None:
         super()._resolve_snrefs(diag_layer)
 
     def get_static_bit_length(self) -> Optional[int]:
-        return getattr(self.diag_coded_type, "bit_length", None)
+        return self.diag_coded_type.get_static_bit_length()
```

Comment:
Drive by?

Reply:
yes. this is too small to factor out, though?

Reply:
include in changelog, important bug fix

Reply:
I put it in a separate PR. I don't think that it currently changes much behavior-wise because the only diag coded type that matters which has the `.bit_length` property/implements `get_static_bit_length()` is `StandardLengthType`...